packages/website/blog/2024-07-31-announcing-typescript-eslint-v8.md

```diff
@@ -496,7 +496,7 @@ To migrate to the new rules:
 - If you were disabling the ban on `{}`, consider enabling [`@typescript-eslint/no-empty-object-type`](/rules/no-empty-object-type), as it allows some cases of `{}` that were previously banned.
 - If you were banning any configurable types lists, provide a similar configuration to [`no-restricted-types`](/rules/no-restricted-types).
 - If you have [`@typescript-eslint/ban-types`](/rules/ban-types) manually enabled, it will no longer ban:
-  - `{}` or `object`: use a [recommended config](/users/configs) or manually enable [`@typescript-eslint/no-empty-object-type`](/rules/no-empty-object-type).
+  - `{}` or `Object`: use a [recommended config](/users/configs) or manually enable [`@typescript-eslint/no-empty-object-type`](/rules/no-empty-object-type).
```

Comment:
Not even `Object`—that would be `no-wrapper-object-types`

Reply:
Oh. 🤔

Reply:
@Josh-Cena ok i change it to just not explicitly mention it at all on the assumption that the reader will most likely either 
- assume it's part of `no-empty-object-type` since it's an uppercase built-in
- click on all the new rule links and find it anyway